common/src/main/java/hunternif/mc/impl/atlas/client/TileTextureMap.java

```diff
@@ -97,6 +94,8 @@ static private Optional<Identifier> guessFittingTextureSet(RegistryKey<Biome> bi
     }
 
     static public Optional<Identifier> guessFittingTextureSetFallback(Biome biome) {
+        /*
```

Comment:
If that doesn´t work anymore because Minecraft changed that, then there´s no reason to keep the code. In fact, the method should be removed instead of stitching useless stuff.

Reply:
Good point! replaced with an `Optional.empty()` expression instead in a61ff06